deploy/olm-catalog/ibm-catalog-ui-operator/3.6.0/ibm-catalog-ui-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -68,6 +68,37 @@ metadata:
               ]
             }
           }
+        },
+        {
+          "apiVersion":"operator.ibm.com/v1alpha1",
+          "kind":"OperandRequest",
+          "metadata":{
+            "name":"catalog-ui-request"
+          },
+          "spec":{
+            "requests":[
+              {
+                "operands":[
+                  {
+                    "name":"ibm-iam-operator"
+                  },                                    
+                  {
+                    "name":"ibm-management-ingress-operator"
+                  },
+                  {
+                    "name":"ibm-platform-api-operator"
+                  },
+                  {
+                    "name":"ibm-helm-api-operator"
+                  },
+                  {
+                    "name":"ibm-helm-repo-operator"
+                  }
```

Comment:
are cert manager and mongodb not dependencies?

Reply:
we only need to list direct dependencies